performance_tests/README.rst

```diff
@@ -0,0 +1,232 @@
+#####################################
+aws-encryption-sdk performance tests
+#####################################
+
+This module runs performance tests for the `AWS Encryption SDK Python`_.
+
+********
+Overview
+********
+
+This module tests the following keyrings / master key-providers:
+
+1. KMS Keyring / KMS Master Key Provider
+2. Raw AES Keyring / AES Master Key Provider
+3. Raw RSA Keyring / RSA Master Key Provider
+4. Hierarchy Keyring
+5. Caching CMM
+
+For each test on the above keyrings / master key-providers, this package measures:
+
+1. Execution time
+2. Total memory consumption
+
+For each keyring / master key-provider, the execution time and memory consumption
+is measured for three operations:
+
+1. Create keyring / master key-provider
+2. Encrypt
+3. Decrypt
+
+The usage of the performance tests is demonstrated through an `AWS KMS Keyring`_.
+However, the procedure is the same for any keyring / master key-provider, with slight
+changes in the input arguments.
+
+The results for the performance test will be available in the results folder in the
+performance_tests directory.
+
+**********************
+Required Prerequisites
```

Comment:
Can you make a note here about the MPL? "Recommended prerequisites"; see https://github.com/aws/aws-encryption-sdk-python/blob/lucmcdon/mpl-deprecations/README.rst

Note that you need MPL to run keyring examples